php-server/Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+pipeline {
+    agent {
+        kubernetes {
+            yaml """
+            apiVersion: v1
+            kind: Pod
+            spec:
+              restartPolicy: Never
+              containers:
+              - name: build-php
+                image: artifactory.cloud.cms.gov/docker/drupal:10.3.0-php8.3-fpm-alpine3.20
+                command: ['tail', '-f', '/dev/null']
+            """
+        }
+    }
+
+    environment {
+        // Initialize default values
+        GIT_MANIFEST_REPOSITORY = ''
+        GIT_MANIFEST_BRANCH = ''
+    }
+
+    stages {
+        stage('Determine Upstream Trigger') {
+            steps {
+                script {
+                    def upstreamProjectCount = 0
+                    currentBuild.upstreamBuilds?.each { b ->
+                        upstreamProjectCount++
+                    }
+                    if (upstreamProjectCount == 0) {
+                        echo "Not an upstream project"
+                    } else {
+                        echo "Upstream project detected"
+                        // Ensure we only set manifest parameters from specific upstream job
+                        currentBuild.upstreamBuilds.each { b ->
+                            if (b.fullDisplayName.contains("Delivery")) {
+                                env.GIT_MANIFEST_REPOSITORY = b.buildVariables.git_manifest_repository ?: params.git_manifest_repository
+                                env.GIT_MANIFEST_BRANCH = b.buildVariables.git_manifest_branch ?: params.git_manifest_branch
+                            }
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
This does not make sense. The Multi Branch Pipeline that runs the steps defined in this Jenkins file *is* the "upstream" project. It invokes the "downstream" SAST, Delivery, and Deployment projects. The Manifest repo URL (`git@github.com:CMS-Enterprise/batcave-knight-light-manifest.git`) should just be hard coded down below.

Reply:
removed the lines.